extensions/src/config/external-tools.js

```diff
@@ -0,0 +1,58 @@
+
+/**
+ * Manage the tools that you can run for specific entity type
+ */
+
+import find from 'lodash/find';
+
+import { getTypeById, generateNotebookParamsObject, getIdByUrl } from '@/tools/utils';
+
+const tmdLink = 'http://morph-service.ocp.bbp.epfl.ch/tmd';
+const blueNaasBase = 'https://blue-naas.humanbrainproject.eu/#/uuid';
+const useCaseWizardBase = 'https://bbp.epfl.ch/public/dev.usecases-wizard/index.html#/entitydashboard';
+
+function generateBlueNaasFullLink(entityId) {
```

Comment:
From here to the bottom should be something that each tool in memodel or emodel knows to manage and not something generic because the links will change depending which entity type is it.